lms/djangoapps/ora_staff_grader/xblock_handlers_mixin.py

```diff
@@ -0,0 +1,79 @@
+""" XBlock handler functions used by ESG endpoints """
```

Comment:
This is probably only ever gonna be used in the views.py, so I might argue that this actually makes more sense in `StaffGraderBaseView`

Reply:
Updated.